src/Sylius/Bundle/ApiBundle/Command/ResetPassword.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+/** @experimental */
+class ResetPassword implements ResetPasswordTokenAwareInterface
```

Comment:
Is `ResetPasswordTokenAwareInterface` necessary? why you can't add it in the command body?
IIRC this token you get from email, so can you explain to me why you choose this way to provide `token` to command?

Reply:
WIll be removed in next iteration